### PR TITLE
change Exitor to exit nft owner (MAT-1996)

### PR DESCRIPTION
**Pull request by AyushKaul**

### CODE REVIEW

File: test/integration/root/ExitScenarios.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const web3Child = utils.web3Child
 chai.use(chaiAsPromised).should()
 let contracts, childContracts, rootERC20, childErc20, statefulUtils
 
-contract('Misc Predicates tests @skip-on-coverage', async function(accounts) {
+contract.only('Misc Predicates tests @skip-on-coverage', async function(accounts) {
```
Comment (imyourm8) [inline test/integration/root/ExitScenarios.test.js:18]: Remove `only`

File: contracts/root/withdrawManager/WithdrawManager.sol [hunk 2]
```diff
@@ -233,11 +233,9 @@ contract WithdrawManager is WithdrawManagerStorage, IWithdrawManager {
             exitQueue.delMin();
             // If the exitNft was deleted as a result of a challenge, skip processing this exit
             if (!exitNft.exists(exitId)) continue;
-
+            address exitor = exitNft.ownerOf(exitId);
+            exits[exitId].owner = exitor;
```
Comment (jdkanani) [inline contracts/root/withdrawManager/WithdrawManager.sol:237]: Do we need to change owner in exists?  @atvanguard @AyushKaul

Comment (AyushKaul) [reply contracts/root/withdrawManager/WithdrawManager.sol:237]: its used while encoding so changed it.
cc: @atvanguard

File: test/integration/root/ExitScenarios.test.js (hunk 1 above)
Comment (AyushKaul) [reply test/integration/root/ExitScenarios.test.js:18]: trying to run tests locally first

**Comment by AyushKaul** [comment]
added test to transfer exit nft before processing the exit.

File: contracts/root/withdrawManager/WithdrawManager.sol (hunk 2 above)
Comment (jdkanani) [reply contracts/root/withdrawManager/WithdrawManager.sol:237]: Do we need it here though?

**Pull request merged**